pkg/commands/objectstorekeys/insert.go

```diff
@@ -0,0 +1,45 @@
+package objectstorekeys
+
+import (
+	"io"
+
+	"github.com/fastly/cli/pkg/cmd"
+	"github.com/fastly/cli/pkg/config"
+	"github.com/fastly/cli/pkg/manifest"
+	"github.com/fastly/cli/pkg/text"
+	"github.com/fastly/go-fastly/v7/fastly"
+)
+
+// InsertCommand calls the Fastly API to insert a key into an object store.
```

Comment:
Similarly this should be `CreateCommand`.

Reply:
Done.